src/pages/ManageApps.vue

```diff
@@ -221,26 +222,19 @@ function ellipsisAppName(appName: string) {
                     <span class="info-detail">{{ paidMausUsed }}</span>
                   </VStack>
                   <VProgressBar
-                    class="info-progress"
-                    style="min-width: 0"
+                    class="info-progress min-w-0"
                     :percentage="100"
                   />
                 </VStack>
               </VStack>
               <VStack class="flex-grow info-margin justify-end">
                 <VStack gap="1.25rem" sm-direction="column">
                   <VStack gap="0.5rem" align="center">
-                    <div
-                      class="legend-dot"
-                      style="background-color: var(--color-green)"
-                    ></div>
+                    <div class="legend-dot bg-[var(--color-green)]"></div>
                     <span class="info-detail-name">Free users</span>
                   </VStack>
                   <VStack gap="0.5rem" align="center">
-                    <div
-                      class="legend-dot"
-                      style="background-color: var(--primary)"
-                    ></div>
+                    <div class="legend-dot bg-[var(--primary)]"></div>
```

Comment:
@svarcoder add these var values to tailwind config and use something like example bg-primary-100 or something

Reply:
Ok.I am doing that.

Reply:
Fix That in All File. Please Check

Reply:
Some Reviews for the Issue as a Whole

1. Remove Every Custom/Hardcoded Style Value in the Tailwind.
2. Update the Tailwind Config up to our requirements to optimise the Styling Code that we write, i.e Height, Padding, Typography and Color Schemes etc
3. Remove The Inline CSS Mods added to the UI Components.
4. Update or Replace the Pre Existing PNGs to modern image formats (SVGs, WebPs)
5. Remove any Redundant or Unused Code or Components.